tasks/main.yml

```
@@ -1,39 +1,45 @@
 ---
 
-- name: Get os specific facts
-  ansible.builtin.include_vars: "{{ item }}"
+- name: get os specific facts
+  ansible.builtin.include_vars: '{{ item }}'
   with_first_found:
     - '{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.yml'
     - '{{ ansible_distribution }}.yml'
     - '{{ ansible_os_family }}.yml'  
```

Needed to fix pipeline:

```suggestion
    - '{{ ansible_os_family }}.yml'
```